build.gradle

```diff
@@ -52,8 +52,8 @@ subprojects {
 
     group = "com.netflix.${githubProjectName}"
 
-    sourceCompatibility = 1.7
-    targetCompatibility = 1.7
+    sourceCompatibility = 1.8
```

Comment:
I would hope a change like this would require a significant version change and not a patch version.